modules/apps/portal-cache/portal-cache-impl/src/main/java/com/liferay/portal/cache/internal/dao/orm/FinderCacheImpl.java

```diff
@@ -502,6 +509,15 @@ protected void activate(BundleContext bundleContext) {
 
 		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
 			bundleContext, ArgumentsResolver.class, "class.name");
+
+		_serviceRegistration = bundleContext.registerService(
```

Comment:
@GabrielHenriquedev plese move this line up, before _serviceTrackerMap, so it follow an alphabetical order.